### ISSUE TITLE
FAIL: test_flatsize_vs_getsizeof (asizeof.test_asizeof.AsizeofTest)

### ISSUE THREAD

**Issue by jlec** [pympler/pympler#22]
```
FAIL: test_flatsize_vs_getsizeof (asizeof.test_asizeof.AsizeofTest)
Test asizeof.flatsize() vs sys.getsizeof()
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/tmp/portage/dev-python/pympler-0.4.2/work/Pympler-0.4.2/test/asizeof/test_asizeof.py", line 82, in test_flatsize_vs_getsizeof
    n, e = test_flatsize(failf=self._failf)  # stdf=sys.stderr)
  File "/var/tmp/portage/dev-python/pympler-0.4.2/work/Pympler-0.4.2/test/asizeof/test_asizeof.py", line 63, in test_flatsize
    a, s, asizeof._nameof(type(o)), _repr(o))
  File "/var/tmp/portage/dev-python/pympler-0.4.2/work/Pympler-0.4.2/test/asizeof/test_asizeof.py", line 76, in _failf
    self.fail(fmt % args)
AssertionError: 904 vs 944 for type: <class 'weakref.KeyedRef'>

----------------------------------------------------------------------
```

with pympler and python2.7

**Comment by rathann** [comment]
Reproduced on Fedora rawhide with python 2.7.10.

**Comment by Pankrat** [comment]
Sorry, I haven't had time to look into this yet. We first need to figure out what's the right size and then work on a fix. In any case, the fallback logic that is tested here is only used on Python 2.5 so it shouldn't keep you from using Pympler on 2.7.

**Comment by Pankrat** [comment]
This is now fixed on master and will be in the upcoming 0.5 release. I've removed the test which was only truly relevant for Python <2.6 anyway.
